nextflow_schema.json

```diff
@@ -483,6 +489,116 @@
                 }
             }
         },
+        "locidex": {
+            "title": "Locidex",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "lx_min_evalue": {
+                    "type": "number",
+                    "default": 0.0001,
+                    "description": "Minimum e-value required for match.",
+                    "minimum": 1e-18,
```

Comment:
Instead of setting a very low, but non-zero minimum I think you can use `"exclusiveMinimum": 0`: https://json-schema.org/understanding-json-schema/reference/numeric#range

I do notice that the [nf-validation docs](https://nextflow-io.github.io/nf-validation/latest/nextflow_schema/nextflow_schema_specification/#numeric-specific-keys) do mention exclusive min/max might not work in all interfaces, though I believe they should work for validating parameters since they are part of the JSON Schema specification (and nf-validation validates against this specification).

This might be something for us to test though (make sure that running mikrokondo triggers an error if you set `--lx_min_evalue 0`).

Reply:
I did not use exclusiveMinimum yet as I did not see it recommended in the Docs section of nf-validation. I did set the minimum value to 0 however. 

updated in: c8e0c8a003520471478e446d9fef5107084608ea

Reply:
Okay, thanks. So long as that minimum value of `0` works then that's great.